### DISCUSSION TITLE
Why do not use GJSON as jsonpath implementation

### DISCUSSION THREAD

**Discussion by skhokhlov** [TwiN/gatus#467] [Ideas]
Current jsonpath implementation is not as powerful as gjson. For example I cannot test keys with dots, but gjson supports it.

https://github.com/tidwall/gjson

**Comment by TwiN** [comment]
Looks like GJSON just created its own custom parsing system, while Gatus _tries_ to follow the JSONPath standard.
Admittedly, a couple of features are missing, but something like allowing keys with dots to be tested should be easy to implement.

Can you give me an example of a condition with a key that has a dot? Just to make sure I understand.
